variables.json

```diff
@@ -12,5 +12,8 @@
   "RUDDER_HEARTBEAT_INTERVAL": "4",
   "RUDDER_NODE_CONFIG_ID": "0",
   "AGENT_RUN_INTERVAL": "5",
-  "CONFIGURATION_REPOSITORY_FOLDER": "/var/rudder/configuration-repository"
+  "CONFIGURATION_REPOSITORY_FOLDER": "/var/rudder/configuration-repository",
+  "STORE_ALL_CENTRALIZED_LOGS_IN_FILE": true,
+  "SYSLOGPORT": "${install_rsyslogd.rsyslog_port[2]}",
+  "SHARED_FILES_FOLDER": "/var/rudder/configuration-repository/shared-files"
```

Comment:
They are used, but it's invisible

STORE_ALL_CENTRALIZED_LOGS_IN_FILE: used in rudder-rsyslog-root.st
SYSLOGPORT: used in promises.st
SHARED_FILES_FOLDER: use in propagatePromises.st